src/main/java/org/apache/sling/jcr/repoinit/impl/DoNothingVisitor.java

```diff
@@ -66,7 +66,7 @@ protected DoNothingVisitor(Session s) {
         session = s;
     }
 
-    protected void report(Exception e, String message) {
+    protected static void report(Exception e, String message) {
```

Comment:
unrelated to the task at hand. please move that to a general cleanup.improvmeent PR.

Reply:
+1

Reply:
i see